xcell/cls/cov.py

```diff
@@ -5,6 +5,7 @@
 import os
 import numpy as np
 import pymaster as nmt
+import time
```

Comment:
do we need this anymore?

Reply:
I wanted to know your thoughts about it. I think it might be interesting to know the numbers in the future. Leaving it here would save us time if we later want to do the timing again for some reason with negligible impact on the performance.